sdk/storage/storage-blob/review/storage-blob.api.md

```diff
@@ -769,6 +769,44 @@ export interface BlobProperties {
     tagCount?: number;
 }
 
+// @public
+export interface BlobQueryCsvTextConfiguration extends BlobQueryTextConfiguration {
+    columnSeparator: string;
```

Comment:
This can't be defaulted to `,`?

Reply:
Let's me confirm from service feature side, if optional is acceptable, let's me make it optional with default values.

Reply:
@bterlson Make it as optional. Please refer to https://github.com/Azure/azure-sdk-for-js/pull/9484